examples/error_estimates_forces.jl

```diff
@@ -0,0 +1,182 @@
+# # Practical error bounds for the forces
```

Comment:
Generally a few more empty lines between comments and code would make the .jl file easier to read.

Reply:
Could you also line-break the long comment lines.

Reply:
Yes, I know some lines are long... I tried to did better in the next commit. However, for the lines with the markdown syntax for list `-` I don't know if there is another way : if I break the line in the comment then the output in the doc is weird...

Reply:
I think if you line-break than you have to indent it appropriately, e.g.
```julia
# - abc def
#   ghi
```
Did you try that? 

Other option is of course to drop the leading `-` ...

Reply:
yep, corrected